src/components/select/select.js

```diff
@@ -116,6 +116,9 @@ angular.module('material.components.select', [
 function SelectDirective($mdSelect, $mdUtil, $mdTheming, $mdAria, $compile, $parse) {
   return {
     restrict: 'E',
+    scope: {
+      selectedText: '=?selectedText'
```

Comment:
Why are you introducing here a new scope? Currently we didn't have any scope, this may break some applications.